### ISSUE TITLE
RTL Support for Drag & Drop (Movable)

### ISSUE THREAD

**Issue by swissspidy** [GoogleForCreators/web-stories-wp#508]
## Feature Description

Follow-up to #55 / #351

When using WordPress admin in RTL context (e.g. by changing your locale or by using the RTL Tester plugin), the X axis is basically flipped for Movable.

If dragging an element 10px to the left, it will be dropped 10px to the right. You can't miss it when testing.

Proposed solution:

Force canvas to be LTR, so that dragging an element to the left actually moves it to the left, and not the opposite.

## Additional Context

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

**Comment by swissspidy** [comment]
As mentioned on #351, an RTL UI does not mean that the document is RTL too.

We could force LTR for the canvas perhaps.

**Comment by miina** [comment]
@swissspidy Would you mind specifying the issues with Movable? (If these are "specifiable").

**Comment by swissspidy** [comment]
@miina The X axis is basically flipped. If dragging an element 10px to the left, it will be dropped 10px to the right. You can't miss it when testing.

**Comment by miina** [comment]
Alright 👍 

Looks like it's not marked as a priority at this moment, otherwise, I'll take a look if/when it becomes a priority.

**Comment by swissspidy** [comment]
Correct, not a priority at this time.

**Comment by swissspidy** [comment]
Moving to backlog as I think it's fairly small to implement and is basically the last big missing issue for proper RTL support.

**Comment by aneeshtan** [comment]
![moving-problem-in-rtl](https://user-images.githubusercontent.com/245936/90213594-7492b680-de0b-11ea-9a09-3f7fbe007713.gif)

This bug is so big that we cannot use the plugin anymore. It would be great if you make changes to fix this issue.

**Comment by swissspidy** [comment]
@o-fernandez @bmattb FYI added the 1.1.0 release to this.

I would really like to  see this sooner rather than later. I don't think it should be too hard to implement, and I want to avoid lots of support questions and negative reviews because we haven't properly implemented RTL support yet.

**Comment by barklund** [comment]
One thing to note: Make sure that rich text input fields (the _draftjs_ components) are still using the global direction rather than being LTR always - so you can still type Arabic and other languages correctly in text fields and they will be correctly aligned.
